### ISSUE TITLE
Runnign VS code on Arch Based Linux distro Manjaro

### ISSUE THREAD

**Issue by KarelDKruger** [pcislo/vscode-nuget-gallery#46]
I am Getting the following error when running the Plugin:
Command 'NuGet: Open NuGet Gallery' resulted in an error (Running the contributed command: 'extension.start' failed.)

**Comment by pcislo** [comment]
Thanks for reporting. I will try to fix it, but I'm not sure when.

**Comment by KarelDKruger** [comment]
Thanks for the response, I will also try it on OpenSUSE and Ubuntu.

**Comment by mbnoimi** [comment]
I get the same error message in Linux Mint 20.1 (I'm unable to use it at all)

```
Command 'NuGet: Open NuGet Gallery' resulted in an error (Running the contributed command: 'extension.start' failed.)
```

**Comment by KarelDKruger** [comment]
> 
> 
> I get the same error message in Linux Mint 20.1 (I'm unable to use it at all)
> 
> **NuGet Gallery v0.0.24**
> 
> ```
> Command 'NuGet: Open NuGet Gallery' resulted in an error (Running the contributed command: 'extension.start' failed.)
> ```

I did test this in Ubuntu and it worked as intended, I recently began suspecting the Kernal version as some rolling releases update that quicker with the main stream Distros always lagging behind, I would however need to defer to the Package author to learn if this may be the case. The snap install of .net also caused some issues recently so I need to sort that out first, but that is a bit off topic for this thread. I hope to get some time this month to do some log digging into this issue for a clearer picture.

**Comment by mbnoimi** [comment]
>  The snap install of .net also caused some issues recently

I installed .net 5.0 as mentioned in the [docs](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu) using apt

**Comment by KarelDKruger** [comment]
> 
> 
> > The snap install of .net also caused some issues recently
> 
> I installed .net 5.0 as mentioned in the [docs](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu) using apt

Because Manjaro is a Arch based Distro its not supported by Microsoft yet so the package is imported by the grate community, but if you want the latest updates from Microsoft unfortunately ,if I want to use Manjaro, I have to go through snap. I will attempt to install a Mint 20.1 partition as well and see if i can recreate the error on mint.

**Comment by mbnoimi** [comment]
> Because Manjaro is a Arch based Distro its not supported by Microsoft yet

What you're talking about?! Linux Mint derived from Ubuntu!!!

**Comment by KarelDKruger** [comment]
> 
> 
> > Because Manjaro is a Arch based Distro its not supported by Microsoft yet
> 
> What you're talking about?! Linux Mint derived from Ubuntu!!!

Yes you are correct, this is a list of supported Distros https://docs.microsoft.com/en-us/dotnet/core/install/linux. The flavor of Linux I am running at this moment is Manjaro which is not supported by Microsoft at this moment in time so I have to follow the manual setup or use the Snap version install method. This unfortunately leaves me in a bind as I need to sort the snap install problem i am having first.

**Comment by ggeorgo** [comment]
Any update/workaround for all of us in Manjaro/Arch to use Nuget Gallery?
I know MS is not officially supporting them, although they are two of the most popular distros, but extensions are supposed to be distro independent...

**Comment by KarelDKruger** [comment]
> Any update/workaround for all of us in Manjaro/Arch to use Nuget Gallery?
> I know MS is not officially supporting them, although they are two of the most popular distros, but extensions are supposed to be distro independent...

I did a clean Manjaro install and only using pacman to install .net first then installed vscode this seems to have solved the issue of nuget gallery for me, but it did cause most of my projects to break. Using this method others may have more luck then me.
I also finally hade time to do a clean install of Mint 20.1 and everything worked fine in Mint for me. It seems to me to be random chance if you get a decent experience. Currently I just opted for the old work around with a windows VM to do any dotnet development as this seems to currently still be the most reliable solution.
